windows/devices/uwb/include/windows/devices/uwb/UwbDeviceConnector.hxx

```diff
@@ -16,17 +16,13 @@
 #include <uwb/protocols/fira/UwbCapability.hxx>
 #include <wil/resource.h>
 #include <windows/devices/uwb/IUwbDeviceDdi.hxx>
+#include <windows/devices/uwb/IUwbSessionDdi.hxx>
 
 namespace windows::devices::uwb
 {
-/**
- * @brief Opaque class forward declaration to help with the deregistration
- *
- */
-class RegisteredCallbackToken;
-
 class UwbDeviceConnector :

```

Comment:
This is a connector for both device and session based DDI functions, so let's rename to `UwbConnector`.
```suggestion
class UwbConnector :
```